prime-angular-frontend/src/app/modules/enrolment/pages/summary/summary.component.html

```diff
@@ -1,7 +1,50 @@
-<div>
+<div class="app-print-layout">
```

Comment:
The `app-print-layout` class is no longer used in the application

Reply:
okay i will remove this